### ISSUE TITLE
Traces field is missing on product creation

### ISSUE THREAD

**Issue by sashabrava** [openfoodfacts/openfoodfacts-androidapp#2994]
The field "Traces" is missing on the tab "Ingredients" when I add new product. For example, "add" page looks like this:
![Screenshot_2020-01-02-22-23-29-672_org openfoodfacts scanner](https://user-images.githubusercontent.com/11160215/71694544-c5d7c380-2daf-11ea-8a89-842115aa09f0.png)

While "edit" page already contains necessary field:
![IMG_20200102_223326](https://user-images.githubusercontent.com/11160215/71694595-edc72700-2daf-11ea-8e0e-f6d0ad1eb5f2.jpg)

Android: 5.1.1
App version: 3.2.3

**Comment by VaiTon** [comment]
@sashabrava You have "fast addition mode enabled". You can disable it by going to settings.

We still need to make it clear that the mode is enabled.

**Comment by teolemon** [comment]
We try to detect traces from the ingredient list, if I remember well. @stephanegigandet

**Comment by sashabrava** [comment]
> We try to detect traces from the ingredient list, if I remember well. @stephanegigandet

Yes, one trace was auto-detected from Polish (eggs), while others (gluten and 'other nuts') no. If this detection works via dictionaries, I can help to expand them.

**Comment by teolemon** [comment]
@sashabrava
You can help at various places:
Ingredients.pm https://github.com/openfoodfacts/openfoodfacts-server/blob/818b71add57fe7ca0c8f85b1ed7dc804d5092c10/lib/ProductOpener/Ingredients.pm#L143
Ingredients translation: https://world-pl.openfoodfacts.org/ingredients?translate=1
Translation of the app/website: https://crowdin.com/project/openfoodfacts/pl

**Comment by teolemon** [comment]
also you can join the Slack: https://slack.openfoodfacts.org
